.github/workflows/build.yml

```diff
@@ -109,7 +109,7 @@ jobs:
           foreach ($nuspecFile in $nuspecFiles){
             nuget pack "$nuspecFile" `
              -Version ${{ env.VERSION }} `
-             -Properties Configuration=${{ env.CONFIG }} `
+             -Properties "Configuration=${{ env.CONFIG }};CopyRightYear=$(Get-Date -Format yyyy)"`
```

Comment:
Is this casing ok? I'd change it to CopyrightYear if it doesn't have any impact on it.

Reply:
yeah my typo :D good spot.